index.html

```diff
@@ -33,28 +53,206 @@
+
+            <label for="phone">Phone:</label>
+            <input type="tel" id="phone" name="phone" autocomplete="tel" />
+          </fieldset>
+
+          <fieldset>
+            <legend>Feedback:</legend>
+
+            <label for="topic">Subject/Topic:</label>
+            <input type="text" id="topic" name="topic" required />
+
+            <label>Feedback Type:</label>
+            <input
+              type="radio"
+              id="suggestion"
+              name="feedback_type"
+              value="suggestion"
+              required
+            />
+            <label for="suggestion">Suggestion</label>
+            <input
+              type="radio"
+              id="concern"
+              name="feedback_type"
+              value="concern"
+            />
+            <label for="concern">Concern</label>
+            <input
+              type="radio"
+              id="appreciation"
+              name="feedback_type"
+              value="appreciation"
+            />
+            <label for="appreciation">Appreciation</label>
+
+            <label for="feedback">Feedback Details:</label><br />
+            <textarea
+              id="feedback"
+              name="feedback"
+              rows="4"
+              cols="50"
+              required
+            ></textarea>
+
+            <label for="priority">Priority Level:</label>
+            <select id="priority" name="priority">
+              <option value="low">Low</option>
+              <option value="medium">Medium</option>
+              <option value="high">High</option>
+            </select>
+
+            <label for="location">Location (if applicable):</label>
+            <input type="text" id="location" name="location" />
+          </fieldset>
+
+          <fieldset>
+            <legend>Follow-up Information:</legend>
+
+            <input type="checkbox" id="followup" name="followup" value="yes" />
+            <label for="followup"
+              >I would like to receive a follow-up response.</label
+            >
+
+            <label for="contact_method">Preferred Contact Method:</label>
+            <select id="contact_method" name="contact_method">
+              <option value="email">Email</option>
+              <option value="phone">Phone</option>
+            </select>
+          </fieldset>
+
+          <fieldset>
+            <legend>Submit Feedback:</legend>
+
+            <input type="checkbox" id="consent" name="consent" required />
+            <label for="consent">I consent to the use of my feedback.</label>
+
+            <input type="submit" value="Submit Feedback" />
+          </fieldset>
+        </form>
+      </section>
+    </main>
+    <footer>
+
+      <h2>Zoro team </h2>
+      <img class ="logo" src="./images/e9070cef1470c8b91916aa1d5160f9d4.jpg"
+          alt="Illustrated graphic of a zoro logo." style="float: left; " />
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
```

Comment:
Nooooo

Don't use <br /> tags to add space. 

Always use CSS for positioning and spacing